compiler/lib/eval.ml

```diff
@@ -170,14 +233,17 @@ type is_int =
   | N
   | Unknown
 
-let is_int info x =
+let is_int ~target info x =
```

Comment:
passing `target` here doesn't look very useful, we use it to check for an invariants that could be moved elsewhere. (e.g. `Code.invariant`)